homeassistant/components/habitica/const.py

```diff
@@ -10,4 +10,5 @@
 SERVICE_API_CALL = "api_call"
 ATTR_PATH = CONF_PATH
 ATTR_ARGS = "args"
+ATTR_DATA = CONF_SERVICE_DATA
```

Comment:
We shouldn't mix service data and event data constants. Please keep them separate.

Reply:
I'm not sure how to do this right, but I'll try anyway.